index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Code Viewer</title>
+</head>
+<body>
+    <header>
+        <nav>
```

Comment:
Hi @tchasinga , here we've done ... 
but I suggest you to give a class to your **nav** that will help you 
to access it easily in your Css file